app/serializers/order_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class OrderSerializer < ActiveModel::Serializer
```

Comment:
Why Paylink_id why not paylink_id. Rename migration for all capital letters always use snack cases